BUILDING.md

```diff
@@ -111,8 +110,8 @@ platforms in production.
   community will only address issues that reproduce on native GNU/Linux
   systems. Issues that only reproduce on WSL should be reported in the
   [WSL issue tracker](https://github.com/Microsoft/WSL/issues). Running the
-  Windows binary (`node.exe`) in WSL is not recommended, and will not work
-  without adjustment (such as stdio redirection).
+  Windows binary (`node.exe`) in WSL is not recommended. It will not work
+  without workarounds.
```

Comment:
The `(such as stdio redirection)` bit is the most significant workaround. IMHO it should stay.

Reply:
Since removing it was an optional nit in the first place, I'll restore it.